assignments/callbacks.js

```diff
@@ -41,25 +41,58 @@ const items = ['Pencil', 'Notebook', 'yo-yo', 'Gum'];
 
 function getLength(arr, cb) {
   // getLength passes the length of the array into the callback.
+  return cb(arr.length);
 }
+function logArrayLength(testis){
+  console.log(`The length of this array is ${testis}`)
+};
+getLength(items,logArrayLength);
 
 function last(arr, cb) {
   // last passes the last item of the array into the callback.
+  return cb (arr[arr.length - 1]);
 }
+function lastItem(test2){
+  console.log(`The last item of the array is ${test2}`);
+};
+last(items,lastItem);
+
 
 function sumNums(x, y, cb) {
   // sumNums adds two numbers (x, y) and passes the result to the callback.
+  let add = x + y;
+  return cb(add); 
 }
+sumNums(5,6, console.log);
 
 function multiplyNums(x, y, cb) {
   // multiplyNums multiplies two numbers and passes the result to the callback.
+  let mul = x * y;
+  return cb(mul); 
 }
+multiplyNums(10,6, console.log);
+
 
 function contains(item, list, cb) {
   // contains checks if an item is present inside of the given array/list.
   // Pass true to the callback if it is, otherwise pass false.
+  //var arr = [1, 2, 3];
+// var evens = arr.filter((num) => {
+//   if (num % 2 === 0) {
+//     return true;
+//   }
+//   return false;
+// });
+  for (let i=0;i<list.length;i++){
+    if(list[i]===item){
+      result=true;
+    }
+    else result = false;
+  }
+  return cb(result);
 }
```

Comment:
It isn't complete, so not really sure if you wanted to continue with this approach, but so far it's a broken one. I've added additional code there for you to play with it (check your console). See if you can find a way to only return `true` if an item is actually present in the list.

`const items = ["Pencil", "Notebook", "yo-yo", "Gum"];

function contains(item, list, cb) {
  for (let i = 0; i < list.length; i++) {
    if (list[i] === item) {
      console.log("This one matches: ", list[i]);
      result = true;
    } else {
      result = false;
      console.log("Doesn't match: ", list[i]);
    }
  }
  return cb(result);
}

contains("Notebook", items, console.log);`

Also, this part could have been solved easier with the help of `includes` method:

`function contains(item, list, cb) {
  if (list.includes(item)) {
    return cb(true);
  } else {
    return cb(false);
  }
}`

Please take a look at [array methods](https://javascript.info/array-methods)